Extension/package.json

```diff
@@ -506,6 +506,15 @@
           "default": "Disabled",
           "description": "If enabled, code is colorized based on IntelliSense. This setting has no effect if IntelliSense is disabled.",
           "scope": "resource"
+        },
+        "C_Cpp.dependencyManager": {
+          "type": "string",
+          "enum": [
+            "vcpkg",
+            "none"
+          ],
+          "default": "vcpkg",
+          "markdownDescription": "Enable integration services for the [vcpkg dependency manager](https://aka.ms/vcpkg/)."
```

Comment:
@michelleangela @bobbrow @tara-raj It would be nice to add more markup to our other settings descriptions to make it easier to read and with more links to docs.

Reply:
Yes! I found it yesterday after a bit of experimenting. It's undocumented AFAIK